app.js

```diff
@@ -36,15 +37,28 @@ function checkParams() {
         } else if (argv.pin && argv.pin.toString().length !== 6) {
             console.error('Pincode must be a 6 digit number \nRefer documentation for more details');
             return;
-        } else if (argv.interval && argv.interval < 5) {
-            console.error('Please provide an interval greater than 5 minutes');
+        } else if (argv.interval && argv.interval < 1) {
+            // these APIs are subject to a rate limit of 100 API calls per 5 minutes per IP
+            console.error('Please provide an interval greater than equal to 2 minutes');
             return;
         } else if (argv.date && !isMatch(argv.date, 'dd-MM-yyyy')) {
             console.error('Please provide date in dd-mm-yyyy format');
             return;
-        } else {
+        } else if (!argv.slot || (argv.slot && argv.slot !== 'dose1' && argv.slot !== 'dose2')) {
```

Comment:
I don't think we should make `slot` a mandatory field. It should work with and without `slot` as argument.

Reply:
If slot is not given, then people will get alert for `dose1` slots even if they want to see `dose2` slots :)